client/src/components/scans/scans.js

```diff
@@ -179,6 +322,30 @@ class Scans extends React.Component {
     return null;
   }
 
+  renderScansActions() {
+    const { viewOptions } = this.props;
+
+    let mergeAllowed = _.size(viewOptions.selectedItems) > 1;
+
+    let deleteAction = null;
+    if (this.okToDelete) {
```

Comment:
this condition resolve? or a placeholder

Reply:
are you ok with just setting okToDelete as a constant or I can remove it. If i remove it I have to remove all the delete handling or comment it alllllll out.

Reply:
Ahhhh it's a placeholder. If we're going to leave it in place then at minimum I think a comment/todo that explains what's happening might be in order since it doesn't appear to resolve. Otherwise, yeahhhh a constant would work, but again a comment should probably be placed in there.